tests/Element/TagBlockTest.php

```
@@ -57,692 +55,675 @@ final class TagBlockTest extends TestCase
+                ->render(),
             "Failed asserting that element renders correctly with 'events()' method.",
         );
     }
 
     public function testRenderWithGlobalDefaultsAreApplied(): void
     {
-        $previous = SimpleFactory::getDefaults(TagBlock::class);
+        SimpleFactory::setDefaults(
+            TagBlock::class,
+            ['class' => 'from-global'],
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)